ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -275,6 +276,12 @@ func ResourceIBMPIInstance() *schema.Resource {
 				Optional:    true,
 				Description: "Custom SAP Deployment Type Information",
 			},
+			PIVirtualOpticalDevice: {
```

Comment:
Why is it not used in Create func?
If only update method supports this parameter then use [this logic](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/ibm/service/power/resource_ibm_pi_instance.go#L404).

Reply:
That is how it was design. Updated to use the logic you pointed out.